modin/core/execution/ray/generic/io/io.py

```diff
@@ -196,3 +197,77 @@ def func(df, **kw):
         for rows in result:
             for partition in rows:
                 wait([partition.oid])
+
+    @staticmethod
+    def _to_parquet_check_support(kwargs):
+        """
+        Check if parallel version of ``to_parquet`` could be used.
+
+        Parameters
+        ----------
+        kwargs : dict
+            Keyword arguments passed to ``.to_parquet()``.
+
+        Returns
+        -------
+        bool
+            Whether parallel version of ``to_parquet`` is applicable.
+        """
+        path = kwargs["path"]
+        compression = kwargs["compression"]
+        if not isinstance(path, str):
+            return False
+        if any((path.endswith(ext) for ext in [".gz", ".bz2", ".zip", ".xz"])):
+            return False
+        if compression is None or not compression == "snappy":
+            return False
+        return True
+
+    @classmethod
+    def to_parquet(cls, qc, **kwargs):
```

Comment:
We should probably add a parallel version of `to_parquet` for Dask engine too since we have parallel `read_parquet` both for Ray and Dask.

Reply:
@YarShev it can happen in a different PR. I think it is probably better. to separate them

Reply:
I don't mind, but the PR title needs to be corrected.